src/scene/mesh.js

```diff
@@ -496,7 +496,46 @@ Object.assign(pc, function () {
         syncAabb: function () {
             // Deprecated
         },
+        destroy:function(){

```

Comment:
Please try to match the coding style of the engine. For this example: destroy: function () { (notice the spaces)

Reply:
Also, leave a black line between the destroy and syncAabb functions.

Reply:
Okay, I'll revise it. I'll pay attention to it next time.